lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -109,6 +109,7 @@ svg.nx-radio-checkbox__control {
   // in FF, the checkboxes don't respect the width: 0 and we need this in order to hide them
   // Fix discovered via the title of https://bugzilla.mozilla.org/show_bug.cgi?id=605985
   -moz-appearance: none;
+  -webkit-appearance: none;
```

Comment:
Does just `appearance` work? Also with this do we still need the `width: 0`?

Reply:
Just having `appearance: none;` doesn't seem to work. And no, it looks like we do not need the `width: 0;` anymore.
19fb2c9f28cc17fde8b1ac0f68650d8f6a026e3a